packages/@aws-cdk/aws-rds/lib/proxy.ts

```diff
@@ -332,7 +355,7 @@ export class DatabaseProxy extends cdk.Resource
     id: string,
     attrs: DatabaseProxyAttributes,
   ): IDatabaseProxy {
-    class Import extends cdk.Resource implements IDatabaseProxy {
+    class Import extends DatabaseProxyBase implements IDatabaseProxy {
```

Comment:
You can now remove the `implements IDatabaseProxy` part:

```suggestion
    class Import extends DatabaseProxyBase {
```